### ISSUE TITLE
Base image update for gs-node-service

### ISSUE THREAD

**Issue by avtar-mindgrep** [godspeedsystems/gs-node-service#632]
Update base image to fix vunerabilities and reduce image size.

**Comment by avtar-mindgrep** [comment]
Need to update the opentelemetry packages for new node version

**Comment by avtar-mindgrep** [comment]
Need to upgrade the open telemetry to support node 16.20.1. Otherwise getting the following error :- 
```
npm notice 
npm notice New major version of npm available! 8.19.4 -> 9.8.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v9.8.0
npm notice Run npm install -g npm@9.8.0 to update!
npm notice 
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: godspeed@1.0.15
npm ERR! Found: @opentelemetry/api@1.4.1
npm ERR! node_modules/@opentelemetry/api
npm ERR!   @opentelemetry/api@"^1.1.0" from the root project
npm ERR!   peer @opentelemetry/api@">=1.0.0 <1.5.0" from @opentelemetry/core@1.15.0
npm ERR!   node_modules/@opentelemetry/core
npm ERR!     @opentelemetry/core@"^1.5.0" from the root project
npm ERR!     @opentelemetry/core@"^1.0.0" from @opentelemetry/instrumentation-express@0.30.0
npm ERR!     node_modules/@opentelemetry/instrumentation-express
npm ERR!       @opentelemetry/instrumentation-express@"^0.30.0" from the root project
npm ERR!   6 more (@opentelemetry/exporter-metrics-otlp-grpc, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! @opentelemetry/sdk-node@"^0.31.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @opentelemetry/api@1.1.0
npm ERR! node_modules/@opentelemetry/api
npm ERR!   peer @opentelemetry/api@">=1.0.0 <1.2.0" from @opentelemetry/sdk-node@0.31.0
npm ERR!   node_modules/@opentelemetry/sdk-node
npm ERR!     @opentelemetry/sdk-node@"^0.31.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/node/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2023-07-18T12_39_14_381Z-debug-0.log
```

**Comment by xCodeMuse** [comment]
@avtar-mindgrep is it meant to go in our current release?
